fastlane/README.md

```diff
@@ -94,7 +94,7 @@ Bundle an iOS sourcemap
 ```
 fastlane ios refresh_dsyms
 ```
-Upload dYSM symbols to Bugsnag from Apple
+Upload dYSM symbols to Sentry from Apple
```

Comment:
This file is auto-generated. You'll need to change that inside of Fastlane instead of in the markdown file.

Reply:
Thanks.

Reply:
Could you please point me to the correct place, @hawkrives?

Reply:
Merge `master` in to resolve this comment. Those lines were _removed_ by an earlier PR. Alternatively, just run `fastlane` locally and it will re-generate this file.